templates/articles.tpl

```diff
@@ -42,6 +42,9 @@
 				{fbvFormSection for="swordPassword" title="user.password"}
 					{fbvElement type="text" password="true" id="swordPassword" value=$swordPassword|escape}
 				{/fbvFormSection}
+				{fbvFormSection for="swordApiKey" title="plugins.importexport.sword.apikey"}
+					{fbvElement type="text" id="swordApiKey" value=$swordApiKey|escape}
```

Comment:
The `fbvElement` Smarty function handles escaping, so `|escape` isn't needed here (and will garble special characters).